dynflow.gemspec

```diff
@@ -24,6 +24,7 @@ Gem::Specification.new do |s|
   s.add_dependency "concurrent-ruby", '~> 1.0'
   s.add_dependency "concurrent-ruby-edge", '~> 0.2.0'
   s.add_dependency "sequel", '>= 4.0.0'
+  s.add_dependency "daemons"
```

Comment:
Is it necessary to make it a hard dependency of dynflow? Wouldn't it be better to put it into a group in Gemfile and make it optional?